ci-operator/config/openshift/debug-network/openshift-debug-network-master.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+  ocp_builder_golang-1.12:
+    name: builder
+    namespace: ocp
+    tag: golang-1.12
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.12:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
```

Comment:
This does not match the RHEL8 builder - you likely want it in sync.

Reply:
done